Civi/Eck/API/Entity.php

```diff
@@ -75,7 +75,7 @@ public function onApi4EntityTypes(GenericHookEvent $event) {
         'searchable' => 'secondary',
         'paths' => [
           'browse' => "civicrm/eck/entity/list/{$entity_type['name']}",
-          'view' => "civicrm/eck/entity?reset=1&action=view&type={$entity_type['name']}&id=[id]",
+          'view' => "civicrm/eck/entity/view?reset=1&action=view&type={$entity_type['name']}&id=[id]",
```

Comment:
I'd like to have the ECK entity tabset open from somewhere, this makes it unreachable from the entity list page. Also, the edit form should then be rendered within the tabset as well, at least that would be more consistent UX-wise.

I'm not sure tabsets are functional within modal popups though.

Reply:
> I'm not sure tabsets are functional within modal popups though.

No they are not. IMO it should be reached from elsewhere, but popups are more convenient from the entity list page & the tabs are therefore not needed in that context.